forced_photometry/multiband_photometry.md

```diff
@@ -6,9 +6,9 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.15.2
 kernelspec:
-  display_name: Python [conda env:tractor]
+  display_name: Python 3 (ipykernel)
   language: python
-  name: conda-env-tractor-py
+  name: python3
```

Comment:
this looks OK and the ones in general work better for generic toolings, but we may need to find a way to ensure that this metadata is not accidentally changed based on the contributor and their workflow.

Reply:
I can't tell if this is an important thing or not.  When I open a notebook, sometimes I get asked which kernel I would like to use, so I click on one, and since kernels change, and I am not perfect, sometimes apparently that action actually gets recorded.  Is this something we can just delete altogether?  Is it something we can configure to not record in the first place?  It seems as though someone wanting to take this code and run it anywhere not on an SP would run into trouble with this current setup?  Or if it just gets ignored, then I am ok to ignore it as well.

Reply:
>  When I open a notebook, sometimes I get asked which kernel I would like to use, so I click on one, and since kernels change, and I am not perfect, sometimes apparently that action actually gets recorded.

That is all good, and you should be able to choose kernels as you like. 

> Is this something we can just delete altogether?

no, the metadata cannot be deleted, and may not cause issues for people taking the notebook (maybe they get a warning if a kernel doesn't exist and it falls back to their default kernel). What we could do is to not commit the changes. 
These ones above are in fact nicer as a more generic metadata is created, so I would like to keep it going fwd.

Reply:
I'll google around, maybe there are config options that would leave metadata changes out of the diff. If there aren't any, then adding changes as patches may work. (either `git add --patch` or using a git GUI, we should figure out a way that would be convenient for you and other team members.)